crates/tokmd-python/tests/test_basic.py

```
@@ -63,14 +63,23 @@ def test_run_json_invalid_mode():
 
 
 def test_run_json_invalid_json():
-    """Test run_json returns error for invalid JSON."""
+    """Test run_json raises ValueError for invalid JSON."""
     import tokmd
 
-    result = tokmd.run_json("lang", "not valid json")
-    data = json.loads(result)
+    with pytest.raises(ValueError) as excinfo:
```

Ruff (0.15.17)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)